### PR TITLE
Documented editorconfig part and also change issue template

### LINKED ISSUE #33: Document how to use the file from #32
**I'm submitting a ...**  (check one with "x")
- [X] bug report

**Current behavior:**

I do not know how to use to file.

**Expected behavior:**

I would like to read in the REAME file about the meaning of the file and the project guidelines which seem to be mainfested in the file.

**Related code:**

#32

### PR DESCRIPTION
**Pull request by S2606**
Fixes #33 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by S2606** [comment]
@niccokunzmann @jajodiaraghav Please review

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 * When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
 * When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
 
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style(Its mandatory).
+* For more information regarding .editorconfig file, see http://editorconfig.org/#download
```
Comment (jajodiaraghav) [inline .github/CONTRIBUTING.md]: Convert `http://editorconfig.org/#download` to a link

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 * When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
 * When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
 
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style(Its mandatory).
```
Comment (jajodiaraghav) [inline .github/CONTRIBUTING.md]: It should be: `code style (It's mandatory)`

File: README.md [hunk 3]
```diff
@@ -58,7 +58,8 @@ When a PR is merged into the master branch, the current badges canbe downloaded.
 
 Contributions, Bug Reports, Feature Requests
 --------------
-This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well. Please report issues in the [GitHub tracker][new-issue].
+This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well. Please report issues in the [GitHub tracker][new-issue].<br>
```
Comment (jajodiaraghav) [inline README.md]: Remove the <br> tag and add a blank line.

File: README.md [hunk 4]
```diff
@@ -58,7 +58,8 @@ When a PR is merged into the master branch, the current badges canbe downloaded.
...
+Also do read [CONTRIBUTING.md](https://github.com/fossasia/badgeyay/blob/development/.github/CONTRIBUTING.md)
```
Comment (jajodiaraghav) [inline README.md]: Also do read -> Also read

Comment (S2606) [reply README.md]: @jajodiaraghav All changes have been made

**Comment by S2606** [comment]
@niccokunzmann @jajodiaraghav Please review

**Comment by jajodiaraghav** [comment]
@S2606 Why did you create so many commits for just one fix? All this should be amended in one commit.
Do you know `git squash`?

**Comment by S2606** [comment]
@jajodiaraghav But since these commits are from Github UI but not cli, would git squash work??

**Comment by jajodiaraghav** [comment]
Just open the command line and perform the squash operation.
Link:
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Also learn to use `git commit --amend`

**Comment by S2606** [comment]
@jajodiaraghav I have a doubt. I have comitted from Github interface not git. So can you please tell me what should i do?

**Comment by jajodiaraghav** [comment]
Nevermind, Will merge this. 

Please learn to use git. Thanks!

**Comment by S2606** [comment]
@jajodiaraghav I am sorry for the mistake.

**Comment by niccokunzmann** [comment]
There is http://try.github.io to learn the command line

**Comment by jajodiaraghav** [comment]
Closed using #43

**Comment by niccokunzmann** [comment]
@jajodiaraghav The goal of collaboration to me is not the code. It is the people working on it or using it.
Creating your own pull-request is fine. I think closing this one is a bit harsh. You can merge and revert or call out what you think does not match guidelines. I value the work of S2606 as I value yours and I do think everyone has an equal right to contribute. There are ways you can go:
1. Close this without a way for S2606 to improve, building up a thought of fault in him which blocks him from further contributing or
2. having a live chat/google-hangout to onboard him to git and github, and have one more motivated contributor.

I think, being a maintainer is mostly maintaining a relationship to the people who contribute.
Here is some inspiration: https://www.youtube.com/watch?v=uj-li0LO_2g
What do you think?

**Comment by jajodiaraghav** [comment]
@niccokunzmann I created a different PR but the commit in it is authored by S2606. All I did was squashing the commits together. Since S2606 is new to FOSS, I felt that asking for a lot of changes could be overwhelming for him and would demotivate him for further involvement. Hence, I did that squash job myself.
I now realize that although my intentions were good, my approach wasn't. I apologize for my mistake.

My primary focus here was to prevent merging bugging and poor code to the codebase.
I will now focus mainly on building better relationship with potential maintainers.
Thanks a lot for your valuable advice :)

**Pull request merged**